cmd/registry-bigquery/README.md

```diff
@@ -0,0 +1,33 @@
+# registry-bigquery
+
+`registry-bigquery` is an experimental command-line tool that uses BigQuery to
```

Comment:
Maybe mention that this only recognizes and processes Open API specs?

Reply:
Right, will do. Thanks!